website/azurerm.erb

```diff
@@ -2224,6 +2224,15 @@
               </ul>
             </li>
 
+            <li>
+              <a href="#">Storage Cache Resources</a>
```

Comment:
rather than making this it's own group could we move this into the Storage list?